test/combineReducers.spec.js

```diff
@@ -237,5 +237,26 @@ describe('Utils', () => {
 
       spy.restore()
     })
+
+    it('only warns for unexpected keys once', () => {
```

Comment:
Right, that should be resolved by using individual cache instances per `combineReducers` call